### ISSUE TITLE
Is ServoRegistry supposed to automatically register to monitors to DefaultMonitorRegistry?

### ISSUE THREAD

**Issue by alechenninger** [Netflix/spectator#511]
I expected to see MBeans start popping up since I'm using the default registry (JmxMonitorRegistry), but as far as I can tell none of my metrics are being registered with the DefaultMonitorRegistry when using the ServoRegistry like so:

```
    MBeanServer mBeanServer = ManagementFactory.getPlatformMBeanServer();
    int mBeanCount = mBeanServer.getMBeanCount();
    Registry registry = new ServoRegistry();
    Timer test = registry.timer("test");
    test.record(10, TimeUnit.SECONDS);
    test.record(3, TimeUnit.SECONDS);
    test.record(5, TimeUnit.SECONDS);
    // Would expect this to fail
    assertEquals(mBeanCount, mBeanServer.getMBeanCount().intValue());
```

Is this expected behavior?

**Comment by alechenninger** [comment]
Similarly, this test passes as well which I would not expect:

```
    Registry registry = new ServoRegistry();
    MonitorRegistry defaultMonitorReg = DefaultMonitorRegistry.getInstance();
    int monitorCount = defaultMonitorReg.getRegisteredMonitors().size();
    Timer test = registry.timer("test");
    test.record(10, TimeUnit.SECONDS);
    test.record(3, TimeUnit.SECONDS);
    test.record(5, TimeUnit.SECONDS);
    assertEquals(monitorCount, defaultMonitorReg.getRegisteredMonitors().size());
```

**Comment by brharrington** [comment]
My guess is that it is related to: https://github.com/Netflix/servo/issues/253

**Comment by brharrington** [comment]
As a side note, if all you care about is exposing to JMX, here is an example using the metrics3 registry:

https://github.com/brharrington/spectator-examples/tree/master/metrics3

**Comment by alechenninger** [comment]
Thank you. I will consider it, but I was hoping I'd get better (as in, each tag getting its own property) out-of-the-box ObjectNames using spectator w/ servo. In the past I've written custom metric namers and parsers with dropwizard (to "get back" the key:values that went into the names) and it works but is a little clumsy.

**Comment by brharrington** [comment]
Is your main target JMX? It would be fairly straightforward to create a spectator JmxRegistry that behaves as you describe. 

We don't really use JMX much for custom metrics for two reasons:

1. A number of our clients have many metrics and registering all these in JMX created strange performance problems that were difficult to debug. In general we found JMX didn't really scale well.
2. Even though the ObjectNames do allow arbitrary properties and let you query them a bit, pretty much all the interfaces I have seen expose them using a tree structure (e.g. JVisualVM or JConsole). So the feedback we got was that they were too cumbersome to use with arbitrary tagging. Out of curiosity what client are you using?
